kuttl/kuttl-test-migrations.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: kuttl.dev/v1beta1
```

Comment:
The files under `kuttl/tests/migrations` are for us to perform smoke tests for the migrations scripts.